source/Core/BSP/Miniware/configuration.h

```diff
@@ -218,6 +218,7 @@
 #define TEMP_NTC             1
 #define ACCEL_I2CBB1         1
 #define POW_EPR              1
+#define TIP_TYPE_SUPPORT     1 // Support for tips of different types, i.e. resistance
```

Comment:
Basically, I try to make the similar approach as for `PROFILE_SUPPORT`: we just enable everything related under the macro.